packages/core/src/_typography.scss

```diff
@@ -281,6 +285,38 @@ Styleguide preformatted
   @extend %code-block;
 }
 
+%keyboard {
```

Comment:
remind me why we do this again? I'd have taken this opportunity to kill this pattern

Reply:
bro i un-killed it for this opportunity! https://github.com/palantir/blueprint/pull/2366/files#diff-63be20ab91cfbb20b47f78f5a125aea1R165

this is a perfect use case for placeholders: one set of styles that we want tied to two selectors - `.X` and `.pt-running-text x`